### DISCUSSION TITLE
can the 'use in drive' save data to a shared loacation?

### DISCUSSION THREAD

**Discussion by nagaisa** [acheong08/Diffusion-ColabUI#39] [Q&A] [answered]
so im wanting to use run in drive for a few resions but counting as its going to take a lot of space and i dont have that, i am wanting to have it run and save in a shared folder (i have another account that's a work/school type with unlimited storage but colab is disabled on it) is there anyway i can do this?

**Comment by acheong08** [comment, marked as answer]
It's better to just not use Drive. The only way you can do this is to run Colab as your other account. Google doesn't allow you to mount the drives of others

**Reply by nagaisa** [reply]
ok
